plugins/CoreVisualizations/javascripts/jqplotEvolutionGraph.js

```diff
@@ -121,6 +122,10 @@
                                 if (period) {
                                     url += '&date=' + date;
                                 }
+
+                                if (segment) {
+                                    url += '&segment=' + encodeURIComponent(decodeURIComponent(segment)); // ensure segment is encoded once
```

Comment:
tested my changes on latest Chrome, Firefox and IE 11. seems to work without any problems. Not sure if that "weird code" is really needed. Might we worth to test it, maybe it could be removed...